apps/hub-app/src/components/Tag.tsx

```diff
@@ -1,17 +1,29 @@
 // COMPONMENT LIBRARY
 
 import styled from 'styled-components';
-import { blueDark } from '@radix-ui/colors';
+import { blueDark, redDark, yellowDark } from '@radix-ui/colors';
```

Comment:
Yes, we're trying to bring in all of the colors in the component library. It is in theme/global/colors I believe when I made the tag component it was a rush to get it in before a demo so I didn't actually create the component-specific alias in our theme yet.

It is some tech debt that we should address.